adafruit_sgp30.py

```diff
@@ -51,7 +51,7 @@
 
 # pylint: disable=bad-whitespace
 _SGP30_DEFAULT_I2C_ADDR  = const(0x58)
-_SGP30_FEATURESET        = const(0x0020)
+_SGP30_FEATURESETS       = [const(0x0020), const(0x0022)]
```

Comment:
ooh ok thats a way to do it -if so, please remove the const()'s and turn this into a tuple ()

Reply:
Ah, ok! I’m pretty clearly not a python guy, plus that was code from 5pm on a Friday. :-\ I now understand that tuples are immutable (and apparently faster) and thus a much better choice here. Will do, thanks! :-)

Reply:
awesome, you're learning :D